src/Faker/Provider/Base.php

```diff
@@ -263,7 +263,7 @@ public static function shuffle($arg = '')
      * @param array $array The set to shuffle
      * @return array The shuffled set
      */
-    public static function shuffleArray($array = array())
+    public static function shuffleArray(array $array = array())
```

Comment:
not at all.

1) in php doc we got: "@param array $array The set to shuffle"
2) in unit tests there is no check for behavior of shuffleArray in case of passing non array there.
3) all usage of shuffleArray -> always pass array into
4) in case of someone tries to pass non array, 

PHP Warning:  reset() expects parameter 1 to be array, string given in php shell code on line 1

will be thrown, - cuz `reset()` is calling on $array var without checking it.

@GrahamCampbell  - please point me on where is BC, - cuz I do not see any.